msteams-platform/apps-in-teams-meetings/in-meeting-notification-for-meeting-stage.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Build in-meeting notification for Teams meeting
+author: v-sdhakshina
+description: In this article, learn how to build in-meeting notification for Microsoft Teams meeting.
+ms.topic: conceptual
+ms.author: v-sdhakshina
+ms.localizationpriority: medium
+---
+
+# Build in-meeting notification for Teams meeting
```

Comment:
This should say more about what an in meeting notification is, and motivate why apps should use them.

Reply:
This page is also a good place to elaborate on the terse NOTEs in the API doc page. We're missing that opportunity.

Reply:
@aosolis differing this for next iteration of revamp

Reply:
Is there a roadmap for the doc revamp that meeting extensibility team is planning? Not clear what's in scope for this change vs future ones. Are we tracking the feedback that's getting deferred to future revisions?